multiplatform-movies/common/src/commonMain/kotlin/com.novoda.movies.gallery/GalleryPresenter.kt

```diff
@@ -0,0 +1,37 @@
+package com.novoda.movies.gallery
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+import kotlin.coroutines.CoroutineContext
+
+class GalleryPresenter internal constructor(private val uiContext: CoroutineContext, private val galleryFetcher: GalleryFetcher) : CoroutineScope {
```

Comment:
Similar to `schedulers` we inject the `CoroutinesContext` to have control over the theading.  For production we use `UI` wherein tests it's `Unconfined` (similar to trampoline).